<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simulador</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(104, 104, 104, 255), stop:1 rgba(216, 216, 216, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>301</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
selection-color: rgb(123, 184, 184);
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="AreaContent1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>299</width>
       <height>359</height>
      </rect>
     </property>
     <widget class="QListWidget" name="listWidget1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>256</width>
        <height>341</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::DoubleClicked</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="addCarpet1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Source/Icons/aíndice.png</normaloff>Source/Icons/aíndice.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>26</width>
      <height>26</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="addFile1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>400</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Source/Icons/}.png</normaloff>Source/Icons/}.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>26</width>
      <height>26</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="delete1">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>400</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(255, 255, 255);
gridline-color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
background-color: rgb(214, 0, 0);
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Source/Icons/asdasd.png</normaloff>Source/Icons/asdasd.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>26</width>
      <height>26</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="copyToSecondTree">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Source/Icons/120827.png</normaloff>Source/Icons/120827.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>78</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="copyToFirstTree">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>170</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Source/Icons/61752.svg</normaloff>Source/Icons/61752.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>70</height>
     </size>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>30</y>
      <width>301</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
selection-color: rgb(123, 184, 184);
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="AreaContent2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>299</width>
       <height>359</height>
      </rect>
     </property>
     <widget class="QListWidget" name="listWidget2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>256</width>
        <height>341</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::DoubleClicked</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="addCarpet2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>400</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Agregar Carpeta&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Source/Icons/aíndice.png</normaloff>Source/Icons/aíndice.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>26</width>
      <height>26</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="addFile2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>400</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Source/Icons/}.png</normaloff>Source/Icons/}.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>26</width>
      <height>26</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="delete2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>400</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(255, 255, 255);
gridline-color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
background-color: rgb(214, 0, 0);
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Source/Icons/asdasd.png</normaloff>Source/Icons/asdasd.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>26</width>
      <height>26</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
